Author: José Manuel Santamaría Lema <panfaust@gmail.com>
Forwarded: no
Description: Set the proper number version, used in the bug reporting assistant.
---
 src/main.cpp |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/src/main.cpp
+++ b/src/main.cpp
@@ -24,7 +24,7 @@
 
 int main(int argc, char *argv[])
 {
-    KAboutData about("kamoso", 0, ki18n(("Kamoso")), "2.0.2", ki18n("Webcam picture retriever"),
+    KAboutData about("kamoso", 0, ki18n(("Kamoso")), "2.0.2+git20130122", ki18n("Webcam picture retriever"),
                 KAboutData::License_GPL, ki18n("(C) 2008-2010 Alex Fiestas and Aleix Pol"));
     about.addAuthor( ki18n("Aleix Pol Gonzalez"), ki18n("Semaphore hacker"), "aleixpol@kde.org" );
     about.addAuthor( ki18n("Alex Fiestas"), ki18n("Coffee drinker"), "alex@eyeos.org" );
